mllib/src/test/scala/org/apache/spark/ml/classification/OneVsRestSuite.scala

```diff
@@ -210,6 +210,19 @@ class OneVsRestSuite extends MLTest with DefaultReadWriteTest {
     assert(ova2.fit(dataset2) !== null)
   }
 
+  test("SPARK-34045: OneVsRestModel.transform should not call setter of submodels") {
```

Comment:
this will fail in master:

![image](https://user-images.githubusercontent.com/7322292/103970929-50fa4f00-51a4-11eb-93e2-24bb2f08ed50.png)